packages/devextreme/js/__internal/grids/grid_core/header_filter/m_header_filter.ts

```diff
@@ -345,7 +346,10 @@ const HeaderFilterController = modules.ViewController.inherit((function () {
               columnsController.columnOption(options.dataField, 'headerFilter', headerFilterByColumn, true);
             });
           },
-          onHidden: () => restoreFocus(this),
+          onHidden: () => {
+            onHidden?.();
```

Comment:
Let's rename inner ```onHidden``` because two ```onHidden``` called inside each other looks very confusing :)
Maybe the ```baseOnHidden``` / ```previousOnHidden``` and etc.
Some name that you like but different from parent ```onHidden```